### PR TITLE
Start screen

### LINKED ISSUE #25: Start screen
Implement a simple start screen for the quiz.

## Implementation
First of all, some preparations. Check out to the main branch and update the project (using git). Then create a branch for keep your work in an isolated environment. 
Then implement the `WelcomeScreen` class into the `screens\welcome_screen.py` file as a subclass of `Screen`. Make sure you import the `WelcomeScreen` at the `screens\__init__.py` file. Call `init_ui()` at the end of the `__init__` method and call `self.ui_process_events(event)` in the end of `process_screen_events` method.
Now, implement the UI. I've created a sample Figma design, try to make something similar to that into the `init_ui` method. You'll find the link in the references below. For the button you need to store a callback during initialization. It just a parameter. To actually fire it, check if the event was equals to a `pygame_gui.UI_BUTTON_PRESSED` in your screen's `process_screen_events` method. If the check is true, call this callback by putting a '()' to the end of the parameter.
Finally, you have one thing left: hook the new screen into the system. Replace the first screen item in the `init_screens` method in the `main.py` script with the new `WelcomeScreen`. Pass the basic params and the `nav_to_first_question` method as your callback without any brackets. Now, write this function: you only have to set `last_screen` to `current_screen` and `current_screen` to `Screens.QUESTION`. Also don't forget to change the `current_state` variable to `Screens.START` at initialization (line 18 when I write this task).

## PR requirements
When you done with all these stuff, make sure that the program is working on your computer and attach a screenshot in the PR.
I may ask you for some minor changes.
If you struggle with something, mention me in the comments below or write me on discord, and we'll figure out for a solution.

## References
- [The Figma Design](https://www.figma.com/file/jFyP8PjFkMOQo1Oew4iyKi/Quiz-Game?type=design&node-id=0%3A1&mode=design&t=ooWaGuzjMlOfElT0-1)
- For a sample solution you can look at the screens in the `answer_screens.py` file.

### PR DESCRIPTION
**Pull request by yefetk**
Bocs a hogy csak most toltom fel de valamiert nem birtam mukodesre birni. Barmit csinaltam mindig valami mas baja volt.

### CODE REVIEW

File: screens/WelcomeScreen.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
+from screens.screen import Screen
+import pygame_gui
+import pygame
+from main import nav_to_question_screen
+
+class WelcomeScreen(Screen):
+    def __init__(self, screen_size, clock, loader):
```
Comment (d0ty) [inline screens/WelcomeScreen.py]: Add set_callback as a constructor param

File: screens/WelcomeScreen.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        super().__init__(screen_size, clock, loader)
+        self.init_ui()
```
Comment (d0ty) [inline screens/WelcomeScreen.py:11]: Add self.set_callback before line 9 and set the set_callback constructor parameter to its value.

**Review by d0ty — CHANGES_REQUESTED** [review]
Please do the following changes and make sure its working. Prove it with a screenshot please.

**Comment by yefetk** [comment]
still not working

**Comment by d0ty** [comment]
Then push it and we'll figure it out what's wrong

**Comment by yefetk** [comment]
ok

**Comment by d0ty** [comment]
I'll test this out tommorrow morning! I hope it will work until our meeting!

**Review by d0ty — APPROVED** [review]

**Pull request merged**